packages/plugin-halson-rest-client/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "@ima/plugin-halson-rest-client",
+	"version": "1.0.1",
+	"description": "HAL+JSON REST API client for IMA applications.",
+	"main": "index.js",
+	"scripts": {
+		"build": "echo \"Release process is supported only via lerna command. See README.md for more info.\"; exit 1",
+		"test": "../../node_modules/.bin/jest --config=jest.config.js",
+		"doc": "../../node_modules/.bin/gulp doc",
+		"prepare": "../../node_modules/.bin/rollup -c"
+	},
+	"keywords": [
+		"ima",
+		"ima.js",
+		"rest",
+		"hal",
+		"json",
+		"halson",
+		"plugin"
+	],
+	"author": "Martin Jurča <martin.jurca@firma.seznam.cz>",
+	"license": "ISC",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/seznam/IMA.js-plugins.git"
+	  },
+	"publishConfig": {
+		"registry": "http://registry.npmjs.org/",
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/seznam/IMA.js-plugins/tree/master/packages/plugin-halson-rest-client/issues"
+	},
+	"homepage": "https://github.com/seznam/IMA.js-plugins/tree/master/packages/plugin-halson-rest-client#readme",
+	"dependencies": {
+		"halson": "3.0.0",
+		"uri-template": "1.0.1"
+	},
+	"devDependencies": {
+	},
+	"peerDependencies": {
+		"@ima/core": "17.x",
+		"@ima/plugin-rest-client": "^3.x"
```

Comment:
You need to specify `@ima/plugin-rest-client` also as in devDependencies, or move it from peerDependencies to dependencies. This way, the tests wont pass, since they are missing the dependencies.